percona/controller/pgcluster/controller_test.go

```diff
@@ -2353,6 +2353,74 @@ var _ = Describe("CR Validations", Ordered, func() {
 			})
 		})
 	})
+
+	Context("Backup repository validations", Ordered, func() {
+		When("creating a CR with valid backup configurations", func() {
+			It("should accept backups disabled with no repositories", func() {
+				cr, err := readDefaultCR("cr-validation-backup-1", ns)
+				Expect(err).NotTo(HaveOccurred())
+
+				cr.Spec.Backups.Enabled = &f
```

Comment:
Hi,
Pls a test case when backups are disabled and there is 1 repo.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1327/commits/51a6d6dc5bcf2783dd7b1bc31ee38e54d014ea93